backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs

```diff
@@ -0,0 +1,81 @@
+using FluentAssertions;
+using Pims.Core.Http;
+using Pims.Core.Test;
+using Moq;
+using System.Diagnostics.CodeAnalysis;
+using Xunit;
+using Microsoft.Extensions.Options;
+using Pims.Ches.Configuration;
+using Pims.Ches;
+using System.Net.Http;
+using Microsoft.Extensions.Configuration;
+using System.Text.Json;
+using System.IdentityModel.Tokens.Jwt;
+using Microsoft.Extensions.Logging;
+using System.IO;
+using System.Text;
+
+namespace Pims.Dal.Test.Libraries.Ches
+{
+    [Trait("category", "unit")]
+    [Trait("category", "ches")]
+    [Trait("group", "notification")]
+    [ExcludeFromCodeCoverage]
+    public class ChesServiceCollectionTest
+    {
+       // public object Configuration { get; private set; }
+
+        public IConfiguration Configuration { get; }
+
+        #region Tests
+        #region ChesServiceCollection Constructors
+        [Fact]
+        public void ChesServiceCollection_Success()
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = PrincipalHelper.CreateForPermission();
+            helper.AddSingleton(user);
+
+            var builder = new ConfigurationBuilder();
+            var options = new ChesOptions()
+            {
+                AuthUrl = "a mocked value",
+                EmailEnabled = true,
+                EmailAuthorized = true
+            };
+            var chesJson = JsonSerializer.Serialize(new { Ches = options });
+            builder.AddJsonStream(new MemoryStream(Encoding.UTF8.GetBytes(chesJson)));
```

Comment:
Can you make sure this `MemoryStream` won't remain open after being used?  They're notorious for memory leaks.

Reply:
What's the best way of disposing of this memory stream? The builder object doesn't have any methods to dispose of the MemoryStream object, would the garbage collection take care of it?